openreview/conference/templates/programchairWebfield.js

```diff
@@ -999,4 +1136,188 @@ $('#group-container').on('click', 'a.collapse-btn', function(e) {
+    $revProgressDiv.html(Handlebars.templates.noteReviewers(reviewerSummaryMap[paperNumber]));
+    updateReviewerContainer(paperNumber);
+
+    var reviewerProfile = {
+      "email": userToAdd,
+      "id": userToAdd,
+      "name": "",
+      "content": {"names": [{"username": userToAdd}]}
+    };
+
+    if (Object.keys(userProfile).length){
+      reviewerProfile = userProfile[userToAdd];
+    }
+
+    conferenceStatusData.profiles[reviewerProfile.id] = reviewerProfile;
+    if (paperNumber in conferenceStatusData.reviewerGroups.byNotes) {
+      conferenceStatusData.reviewerGroups.byNotes[paperNumber][nextAnonNumber] = reviewerProfile;
+    } else {
+      conferenceStatusData.reviewerGroups.byNotes[paperNumber] = { nextAnonNumber : reviewerProfile };
+    }
+    if (reviewerProfile.id in conferenceStatusData.reviewerGroups.byReviewers) {
+      conferenceStatusData.reviewerGroups.byReviewers[reviewerProfile.id].push(paperNumber);
+    } else {
+      conferenceStatusData.reviewerGroups.byReviewers[reviewerProfile.id] = [paperNumber];
+    }
+
+
+    promptMessage('Email has been sent to ' + view.prettyId(userToAdd) + ' about their new assignment to paper ' + paperNumber);
+    var postData = {
+      groups: [userToAdd],
+      subject: SHORT_PHRASE + ": You have been assigned as a Reviewer for paper number " + paperNumber,
+      message: 'This is to inform you that you have been assigned as a Reviewer for paper number ' + paperNumber +
+      ' for ' + SHORT_PHRASE + '.' +
+      '\n\nTo review this new assignment, please login and click on ' +
+      'https://openreview.net/forum?id=' + paperForum + '&invitationId=' + getInvitationId(OFFICIAL_REVIEW_NAME, paperNumber.toString()) +
+      '\n\nTo check all of your assigned papers, please click on https://openreview.net/group?id=' + REVIEWERS_ID +
+      '\n\nThank you,\n' + SHORT_PHRASE + ' Area Chair'
+    };
+    return Webfield.post('/mail', postData);
+  });
+  return false;
+});
+
+$('#group-container').on('click', 'a.unassign-reviewer-link', function(e) {
+  var $link = $(this);
+  var userId = $link.data('userId');
+  var paperNumber = $link.data('paperNumber');
+  var reviewerNumber = $link.data('reviewerNumber');
+
+  Webfield.delete('/groups/members', {
+    id: CONFERENCE_ID + '/Paper' + paperNumber + '/Reviewers',
+    members: [userId]
+  })
+  .then(function(result) {
+    return Webfield.delete('/groups/members', {
+      id: CONFERENCE_ID + '/Paper' + paperNumber + '/AnonReviewer' + reviewerNumber,
+      members: [userId]
+    });
+  })
+  .then(function(result) {
+    currentReviewerToPapersMap = conferenceStatusData.reviewerGroups.byReviewers[userId];
+
+    if (currentReviewerToPapersMap.length === 1) {
+      delete conferenceStatusData.reviewerGroups.byReviewers[userId];
+    } else {
+      conferenceStatusData.reviewerGroups.byReviewers[userId] = conferenceStatusData.reviewerGroups.byReviewers[userId].filter(function(number) {
```

Comment:
You could replace `conferenceStatusData.reviewerGroups.byReviewers[userId]` on the right side of the equals with `currentReviewerToPapersMap`